lib/plugins/features/movies/controller.js

```diff
@@ -3,7 +3,6 @@
 const Movie = require('../../../models/movie');
 
 exports.create = (payload) => {
-  const payloadWithName = Object.assign({ name: payload.title }, payload);
-  return new Movie().save(payloadWithName)
+  return new Movie().save(payload)
```

Comment:
this payload should just be `name` and `release_year` but since were gonna be leaving the validator the same, we need to do additional stuff here